package.json

```diff
@@ -4,94 +4,82 @@
   "repository": "https://github.com/WoWAnalyzer/WoWAnalyzer",
   "license": "AGPL-3.0-or-later",
   "dependencies": {
```

Comment:
This includes all libraries we actually use in the app. I moved the libraries that are just their for the dev environment to the devDependencies.